src/test/java/org/example/result/ConsoleResultViewerTest.java

```diff
@@ -0,0 +1,61 @@
+package org.example.result;
+
+import org.example.model.question.Question;
+import org.example.model.question.Test;
+import org.example.model.user.Student;
+import org.junit.jupiter.api.BeforeEach;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+class ConsoleResultViewerTest {
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    @Mock
+    private Student student;
+
+    @Mock
+    private Test test;
+
+    @Mock
+    private Question question;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Try to avoid using deprected methods:  'initMocks(java.lang.Object)' is deprecated